### ISSUE TITLE
Torch script support for Compose, RandomApply and others

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/vision#2641]
## 🚀 Feature

Idea is to make sure that a compose of transforms could be scripted. For example, the following should work
```python
transforms = T.Compose([
    T.RandomResizedCrop(...),
    T.Random.RandomHorizontalFlip(),
    T.Normalize(...)
])

transforms = torch.jit.scrtipt(transforms)

img = torch.Tensor(...)
out_img = transforms(img)
```

Transforms to support:
- [ ] Normalize
- [ ] Compose
- [ ] RandomApply

cc @vfdev-5

**Comment by vfdev-5** [comment]
We have to go progressively with that. Let's suggest to the user to use `torch.nn.Sequential` in order to torch jit script transformations.

**Comment by bgswaroop** [comment]
What is the difference between "Scripted transforms" and the transforms that are "composed". The only difference as per the docs is that scripted transforms support only Torch tensors. Is there any other difference or benefit of using one over the other?

**Comment by fmassa** [comment]
@bgswaroop scripted transforms allow you to use them in C++ via torchscript, while other transforms don't and you'd have to re-implement them in C++ (by for example using OpenCV)
